cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -408,21 +407,35 @@ public String parseLogging(Logging logging, ConfigMap externalCm) {
                 newSettings.addMapPairs(inlineLogging.getLoggers());
             }
 
-            return createPropertiesString(newSettings);
-
+            return createLog4jProperties(newSettings);
         } else if (logging instanceof ExternalLogging) {
             if (externalCm != null && externalCm.getData() != null && externalCm.getData().containsKey(getAncillaryConfigMapKeyLogConfig())) {
-                return externalCm.getData().get(getAncillaryConfigMapKeyLogConfig());
+                return addMonitorIntervalToExternalLogging(externalCm.getData().get(getAncillaryConfigMapKeyLogConfig()));
             } else {
                 log.warn("ConfigMap {} with external logging configuration does not exist or doesn't contain the configuration under the {} key. Default logging settings are used.",
                         ((ExternalLogging) getLogging()).getName(),
                         getAncillaryConfigMapKeyLogConfig());
-                return createPropertiesString(getDefaultLogConfig());
+                return createLog4jProperties(getDefaultLogConfig());
             }
 
         } else {
             log.debug("logging is not set, using default loggers");
-            return createPropertiesString(getDefaultLogConfig());
+            return createLog4jProperties(getDefaultLogConfig());
+        }
+    }
+
+    /**
+     * Adds 'monitorInterval=5' to external logging ConfigMap. If ConfigMap already has this value, it is persisted.
+     * This property is ignored by log4j but used by log4j2.
```

Comment:
Does this mean we are adding this to both Log4j2 logs where it is used as well as to log4j logs where it is ignored? That would be wrong since it would be confusing to anyone who checks the map. You should add it only in the place where it is supported.